cmd/hubagent/main.go

```diff
@@ -59,6 +60,7 @@ func init() {
 	utilruntime.Must(fleetv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(workv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(placementv1beta1.AddToScheme(scheme))
+	utilruntime.Must(placementv1alpha1.AddToScheme(scheme))
```

Comment:
should we not include the new apis in the main func? until the crd release has been finished.